xgcm/autogenerate.py

```diff
@@ -272,3 +285,28 @@ def fill_attrs(da, pos, axis):
 
     da.attrs = attrs
     return da
+
+
+def dll_dist(dlon, dlat, lon, lat):
+    """Converts lat/lon differentials into distances
+
+    PARAMETERS
+    ----------
+    dlon : xarray.DataArray longitude differentials
+    dlat : xarray.DataArray latitude differentials
+    lon  : xarray.DataArray longitude values
+    lat  : xarray.DataArray latitude values
+
+    RETURNS
+    -------
+    dx  : xarray.DataArray distance inferred from dlon
+    dy  : xarray.DataArray distance inferred from dlat
+    """
+
+    # First attempt with super cheap approach...
+    #     111km for each deg lat and then scale that by cos(lat) for lon
+    dll_factor = 111000.0
+    dx = dlon * np.cos(np.deg2rad(lat.data)) * dll_factor
+    dy = dlat * dll_factor
+
+    return dx, dy
```

Comment:
This function looks useful, but why is it in this PR? It seems like a new, separate feature.